### ISSUE TITLE
generated nql property’s value has no ""   and   SyntaxError: syntax error near `execute'

### ISSUE THREAD

**Issue by 0914ds** [nebula-contrib/graph-ocean#23]
**my cluster‘s nebulaGraph version is 3.5, and i use this java client to operate nebula.**
two issues show up:
1, the property'value after "set" has no ""  in the generated nql ,so i try add  following code before contact string nql,   like this:
```
 if (entry.getValue()` instanceof  String){
                entry.setValue("\""+entry.getValue()+"\"");
            }
```
this code added in NebulaBatchEdgesUpdate.java and NebulaBatchVertexUpdate.java  and new generated nql can be execute in web consol.

2，after i add above code , another error occurs
`SyntaxError: syntax error near `execute'`

anyone can fix it ?

**Comment by Anyzm** [comment]
感谢您的来件，邮件已收到。

**Comment by wey-gu** [comment]
I see you tried to add `"` for string-typed props but still encountering issues, could you print the final nGQL line?

I think of two issues:

- we dont need `user.user_name` but `user_name`, see below from docs

```
UPSERT VERTEX ON player "player666"
        SET age = 31 
        WHEN name == "Joe" 
        YIELD name AS Name, age AS Age;
```

- `user` is a keyword, we better escape all tokens like this to avoid this:

```
UPSERT VERTEX ON `user` "player666"
        SET `age` = 31 
        WHEN `name` == "Tom" 
        YIELD `name` AS `Name`, `age` AS `Age`;
```

After above changes confirmed to be correct, would you mind create a PR to improve this project accordingly?

Thanks!

**Comment by 0914ds** [comment]
thanks wey, it belongs to graph-ocean，i post it in wrong place

**Comment by Anyzm** [comment]
> thanks wey, it belongs to graph-ocean，i post it in wrong place
hi，Which version of GraphOcean are you using?
I remember this issue being resolved in versions 2.0.0-rc1 and 3.0.1.

**Comment by Anyzm** [comment]
> thanks wey, it belongs to graph-ocean，i post it in wrong place

I have now merged the 3.0.1 code into mster
